.github/workflows/build-kernel-5.10.yml

```diff
@@ -102,6 +102,12 @@ jobs:
         echo "[+] Add KernelSU symbols"
         cat $KSU_ROOT/kernel/export_symbol.txt | awk '{sub("[ \t]+","");print "  "$0}' >> $SYMBOL_LIST
 
+    - name: Set boot sign key
```

Comment:
What is this for? better move to a single pr

Reply:
e, bad merge